torch/csrc/autograd/record_function_ops.cpp

```diff
@@ -67,6 +57,7 @@ c10::intrusive_ptr<c10::ivalue::Future> _call_end_callbacks_on_fut(
 
 // Internal only, do not use directly, use Python's record_function()
 TORCH_LIBRARY_FRAGMENT(profiler, m) {
+    m.class_<at::RecordFunction>("RecordFunction");
```

Comment:
nit: `_RecordFunction` as it's not meant to be used publicly?

Reply:
Updated.